spec/support/rails_template.rb

```diff
@@ -26,7 +26,7 @@ class Post < ActiveRecord::Base
       # nothing to see here
     end
 
-    unless Rails::VERSION::MAJOR > 3 && !defined? ProtectedAttributes
+    unless !defined? ProtectedAttributes
       attr_accessible :id, :title, :body, :starred, :author, :position, :published_date, :author_id, :custom_category_id, :category
     end
```

Comment:
Wrong line indent and obsolete `end`, same for the next 5 `unless ...` removements